charts/cluster/values.yaml

```diff
@@ -153,6 +153,15 @@ cluster:
   enableSuperuserAccess: true
   superuserSecret: ""
 
+  # -- This feature automatically manages physical replication slots for each hot standby replica in
+  # the High Availability cluster, both in the primary and the standby.
+  # See: https://cloudnative-pg.io/documentation/current/replication/#replication-slots-for-high-availability
+  replicationSlots:
+    highAvailability:
+      enabled: true
+      slotPrefix: _cnpg_
+      updateInterval: "30"
```

Comment:
Can you replace this with:

```
  replicationSlots: {}
   # highAvailability:
   #   enabled: true
   #   slotPrefix: _cnpg_
   #   updateInterval: "30"
```

and then re-run `make schema docs`.

Otherwise the generated schema is incorrect and would not allow otherwise valid configurations. This way it would only validate whether it's passed an object.

Reply:
Sorry, I not get a bit what you mean. This will be this way, what else can be set outside of this schema?

Reply:
Refer to the API reference here: https://cloudnative-pg.io/documentation/current/cloudnative-pg.v1/#postgresql-cnpg-io-v1-ReplicationSlotsConfiguration

Reply:
The problem is that our schema is auto generated based on values, so it will assume the structure you provided, which is incomplete. See `make schema`.

Reply:
I found and fixed issue, thanks